### PR TITLE
repository moved to github

### PR DESCRIPTION
**Pull request by stefanil**

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        archivedFileCount: 7
+        timeZone: CET
+      - type: file
+        threshold: DEBUG
+        currentLogFilename: ../logs/energymanager-debug.log
+        archivedLogFilenamePattern: ../logs/energymanager-debug-%d{yyyy-MM-dd-HH-mm}.log.gz
+        archivedFileCount: 3
+        timeZone: CET
+database:
+#  driverClass: org.hsqldb.jdbc.JDBCDriver
```
Comment (mthiele) [inline config.yml]: Raus.

File: dependency-reduced-pom.xml [hunk 2]
```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mthiele) [inline dependency-reduced-pom.xml:1]: .gitignore

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+      <groupId>com.h2database</groupId>
+      <artifactId>h2</artifactId>
+      <version>1.4.192</version>
+    </dependency>
+    <dependency>
+      <groupId>org.modelmapper</groupId>
+      <artifactId>modelmapper</artifactId>
+      <version>0.7.5</version>
+    </dependency>
+    <!--<dependency>-->
```
Comment (mthiele) [inline pom.xml]: Raus.

File: src/main/java/de/saxsys/energymanager/DatabaseConfiguration.java [hunk 4]
```diff
@@ -0,0 +1,79 @@
+/*
```
Comment (mthiele) [inline src/main/java/de/saxsys/energymanager/DatabaseConfiguration.java]: Hust.

File: src/main/java/de/saxsys/energymanager/EnergyManagerApplication.java [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+  }
+
+  @Override
+  public String getName() {
+    return "EnergyManager";
+  }
+
+  @Override
+  public void initialize(final Bootstrap<EnergyManagerConfiguration> bootstrap) {
+    // TODO: application initialization
```
Comment (mthiele) [inline src/main/java/de/saxsys/energymanager/EnergyManagerApplication.java]: Raus.

File: src/main/java/de/saxsys/energymanager/db/DbUtil.java [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+      }
+
+      return function.apply(CONNECTION);
+    } catch (final SQLException exc) {
+      LOG.error("SQL error on database operation.", exc);
+    } finally {
+      try {
+        if (CONNECTION != null) {
+          CONNECTION.close();
+//          CONNECTION = null;
```
Comment (mthiele) [inline src/main/java/de/saxsys/energymanager/db/DbUtil.java]: Raus.

File: src/main/java/de/saxsys/energymanager/db/SolarPanelMonitor.java [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+import java.util.ArrayList;
+import java.util.concurrent.ThreadLocalRandom;
+
+import javax.inject.Inject;
+import javax.inject.Provider;
+import javax.inject.Singleton;
+import javax.persistence.EntityManager;
+
+@Singleton
+public class SolarPanelMonitor {
```
Comment (mthiele) [inline src/main/java/de/saxsys/energymanager/db/SolarPanelMonitor.java]: Dao?

File: src/test/resources/it-config.yml [hunk 8]
```diff
@@ -0,0 +1,7 @@
+logging:
+  level: DEBUG
+  appenders:
+      - type: console
+        timeZone: CET
+database:
+  connected: false
```
Comment (mthiele) [inline src/test/resources/it-config.yml]: Weg?

File: config.yml (hunk 1 above)
Comment (stefanil) [inline config.yml]: Done.

File: dependency-reduced-pom.xml (hunk 2 above)
Comment (stefanil) [inline dependency-reduced-pom.xml:1]: Done.

File: pom.xml (hunk 3 above)
Comment (stefanil) [inline pom.xml]: Done.

File: src/main/java/de/saxsys/energymanager/DatabaseConfiguration.java (hunk 4 above)
Comment (stefanil) [inline src/main/java/de/saxsys/energymanager/DatabaseConfiguration.java]: Für alle: Done.

File: src/main/java/de/saxsys/energymanager/EnergyManagerApplication.java (hunk 5 above)
Comment (stefanil) [inline src/main/java/de/saxsys/energymanager/EnergyManagerApplication.java]: Done.

File: src/main/java/de/saxsys/energymanager/db/DbUtil.java (hunk 6 above)
Comment (stefanil) [inline src/main/java/de/saxsys/energymanager/db/DbUtil.java]: Done.

File: src/main/java/de/saxsys/energymanager/db/SolarPanelMonitor.java (hunk 7 above)
Comment (stefanil) [inline src/main/java/de/saxsys/energymanager/db/SolarPanelMonitor.java]: Done.

File: src/test/resources/it-config.yml (hunk 8 above)
Comment (stefanil) [inline src/test/resources/it-config.yml]: Korrigiert. Done.

**Pull request merged**